_includes/menu.html

```diff
@@ -2,7 +2,7 @@
 <header class="navbar navbar-default navbar-demo navbar-fixed-top eslint-nav" id="top" role="banner">
   <div class="container">
 
-    <a href="/" class="navbar-brand"><img alt="ESLint" src="/img/logo.svg" itemprop="image">ESLint</a>
+    <a href="{{ site.url }}/" class="navbar-brand"><img alt="ESLint" src="{{ site.url }}/img/logo.svg" itemprop="image">ESLint</a>
```

Comment:
Ah, got it. Thanks. As I mentioned, I always used local Jekyll, and that's not a problem there, since it's always `http://localhost:port/` in that case, I'm 👍 for adding that in.